### PR TITLE
(feature request) net451

### PR DESCRIPTION
**Pull request by viceice**
It would be nice to use this package for net451 and above tests.

Regards
Michael

### CODE REVIEW

**Comment by viceice** [comment]
I can add an appveyor file to build on windows.

File: .travis.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ before_install:
 script:
   - dotnet restore || travis_terminate 1
   - dotnet pack --no-restore src/JunitTestLogger/JUnitTestLogger.csproj --configuration Release /p:Version=$(./gogitver) || travis_terminate 1;
-  - dotnet nuget push src/JunitTestLogger/bin/Release/*.nupkg -s https://www.nuget.org/api/v2/package -k $NUGET_API_KEY || travis_terminate 1;
\ No newline at end of file
+  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then dotnet nuget push src/JunitTestLogger/bin/Release/*.nupkg -s https://www.nuget.org/api/v2/package -k $NUGET_API_KEY || travis_terminate 1; fi'
```
Comment (thzinc) [inline .travis.yml:12]: Good change. Thanks.

**Comment by thzinc** [comment]
@ViceIce, thank you for this contribution! I do have some concerns around longer-term maintainability of adding additional target frameworks. As an alternative, is it possible to change the target framework to `netstandard1.2` in order to gain compatibility with .NET Framework 4.5.1? (Reference: [.NET Standard implementation support](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support))

File: .travis.yml (hunk 1 above)
Comment (viceice) [reply .travis.yml:12]: 👍

**Comment by viceice** [comment]
Maybe, but my time is also very limited. 😄 

Ok, i will update this pr.

**Comment by viceice** [comment]
I've looked at [Microsoft.TestPlatform.ObjectModel](https://www.nuget.org/packages/Microsoft.TestPlatform.ObjectModel/15.5.0) which is only available for net451, netstandard1.4 and netstandard1.5. That are the only target frameworks we can use.

**Comment by viceice** [comment]
Maybe we should build for netstandard1.5 and net451 like the trx logger 5.9.0

https://github.com/Microsoft/vstest/blob/25fb7c25207d93e277a55c9bb800da83f9029fca/src/Microsoft.TestPlatform.Extensions.TrxLogger/Microsoft.TestPlatform.Extensions.TrxLogger.csproj#L10

**Comment by viceice** [comment]
I no longer use this.

**Pull request closed without merging**